File: Labs/Lab2/0131-Lab2.java
import java.util.Arrays;

public class Main {
    public static void main(String[] args) {
        String sentence = "secure";
        String tsentence = "rescue";
        if (isAnagram(tsentence, sentence)) {
            System.out.println("The two strings are anagrams");
        } else {
            System.out.println("Not anagrams");
        }//else
    }//main

    public static boolean isAnagram(String t, String s) {
        char[] tArray = t.toCharArray();
        char[] sArray = s.toCharArray();
        Arrays.sort(tArray);
        Arrays.sort(sArray);
        return Arrays.equals(tArray, sArray);
    }//isanagram
}//main

File: Labs/Lab2/0131-Lab2.py
def anagram(t, s):
    t_hm = {} #initialize empty hashmap for t
    s_hm = {} #initialize empty hashmap for s
    for character in t:
        if character in t_hm:
            t_hm[character] += 1
        else:
            t_hm[character] = 1
    for character in s:
        if character in s_hm:
            s_hm[character] += 1
        else:
            s_hm[character] = 1

    return(t_hm == s_hm)

sentence = "secure"
tsentence = "rescue"
if anagram(tsentence, sentence):
    print("the two strings are anagrams")
else:
    print("not anagrams")
